Fix output location for the HTML result dump

firepol opened the pull request:
Trade results were saved in the simulations/ folder. To avoid confusion,
paper results are prefixed with "paper" (& still saved in simulations/).

DeviaVir commented on commands/trade.js:
```diff
@@ -217,16 +217,16 @@ module.exports = function container (get, set, clear) {
             if (so.filename !== 'none') {
               var out_target
-              
+              var out_target_prefix = so.paper ? 'simulations/paper_result_' : 'stats/trade_result_'
```
`stats/trade_result`?

firepol replied:
`trade_result_` was already the prefix of the file name. Stats, because I was checking the stats folder: https://github.com/DeviaVir/zenbot/tree/unstable/stats

The readme says *HTML Outputs from trades will be placed here!*
I've run several live modes and they were saved in the simulations folder, which doesn't make sense. They should be saved in the stats folder (even he readme says so, or did I misunderstood something?).

I added the variable `out_target_prefix` which is used a few lines below.

Actually if you are ok with this change, I forgot to add paper_result* in the .gitignore:

Before:

```
sim_result*
trade_result*
```

After:

```
sim_result*
trade_result*
paper_result*
```

I'm still new to github pull requests. For such a thing what should I do? Close this pull request and create a new one that includes also the .gitignore update so we have just one commit with everything together?

dauv replied:
@firepol Regarding your pullrequest question. If you want everything in a single commit. You can do the following:

First reset the HEAD (add a ^ for each commit you want to reset)  of your local branch:
```
git reset HEAD^
```

Then make your change and commit again. And force push to your remote branch, to overwrite the commtis in the pullrequest:

```
git push stats --force
```

firepol replied:
Thanks @dauv, I added `paper_result*` to the gitignore, reset my commit and forced pushed to it, github automatically updated it. Cool stuff.

Commit:
Fix output location for the HTML dump

Trade results were saved in the simulations/ folder. To avoid confusion,
paper results are prefixed with "paper" (& still saved in simulations/).
